app/assets/javascripts/select-kit/addon/components/composer-actions.js

```diff
@@ -30,30 +30,34 @@ export default DropdownSelectBoxComponent.extend({
   pluginApiIdentifiers: ["composer-actions"],
   classNames: ["composer-actions"],
   isEditing: equal("action", EDIT),
+  isInSlowMode: gt("topic.slow_mode_seconds", 0),
 
   selectKitOptions: {
     icon: "iconForComposerAction",
     filterable: false,
     showFullTitle: false,
   },
 
-  iconForComposerAction: computed("action", "whisper", "noBump", function () {
-    if (this.isEditing) {
+  @discourseComputed("isEditing", "action", "whisper", "noBump", "isInSlowMode")
+  iconForComposerAction(isEditing, action, whisper, noBump, isInSlowMode) {
+    if (isEditing) {
       return "pencil-alt";
-    } else if (this.action === CREATE_TOPIC) {
+    } else if (action === CREATE_TOPIC) {
       return "plus";
-    } else if (this.action === PRIVATE_MESSAGE) {
+    } else if (action === PRIVATE_MESSAGE) {
       return "envelope";
-    } else if (this.action === CREATE_SHARED_DRAFT) {
+    } else if (action === CREATE_SHARED_DRAFT) {
       return "far-clipboard";
-    } else if (this.whisper) {
+    } else if (whisper) {
       return "far-eye-slash";
-    } else if (this.noBump) {
+    } else if (noBump) {
       return "anchor";
+    } else if (isInSlowMode) {
```

Comment:
I wonder, what happens if you are editing a post that is in a slow mode topic?

Reply:
It's a good catch! We should show the slow mode indicator when editing a post. 

In other cases, we shouldn't show it since they either aren't affected by slow mode restrictions (like when a user answering with the new topic) either intended to be used by staff (like when answering with a whisper).